en/collect/import-using-online-bibliographic-database/ccsb.md

```diff
@@ -0,0 +1,3 @@
+# Collection of Computer Science Bibliographies
+
+The [Collection of Computer Science Bibliographies](https://liinwww.ira.uka.de/bibliography/) is a public search engine for bibliographies of scientific literature in computer science.
```

Comment:
This is OK.

Other descriptions are longer  (see, eg. https://github.com/JabRef/user-documentation/blob/master/en/collect/import-using-online-bibliographic-database/gvk.md)

Proposal: Quote the homepage using `>`

> You may use Lucene syntax, available fields are: ti (title), au (author), yr (publications year). In short: the default operator is 'OR', use '+' and '–' operators to specify ('+') required and ('–') forbidden words. Wildcards: '?' – single letter, and '*' – part of the word (at the end of a word), '~' – specified at the end of a word makes approximate match (allows typos, but takes many seconds to process)

Reply:
@koppor what do you think about adding all fetchers under https://docs.jabref.org/collect/import-using-online-bibliographic-database as a big list, each with a short description and a longer `<details>` pane (for advanced search). It feels a bit strange to have a page for each fetcher with one or two sentences.

Reply:
Yeah, sounds good!